src/adapters/indexExchange.js

```diff
@@ -62,29 +340,26 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
 						if (currentId === adSlotId) {
 							var bidObj = slotIdMap[adSlotId];
 							var adUnitCode = bidObj.placementCode;
-							var slotObj = getSlotObj(lookupObj, adSlotId);
+							var slotObj = getSlotObj(cygnus_index_args, adSlotId);
+
 							bid = bidfactory.createBid(1);
-							bid.cpm = (currentCPM / 100);
+							bid.cpm = currentCPM / 100;
 							bid.ad = adContents;
```

Comment:
@headertag 

I noticed `adContents` is actually an array with a single ad response. Is it always a single size array, and should we always take the first element from it?